src/backend/app.js

```diff
@@ -21,10 +22,114 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//app.get("/my-route", (req, res) => {
+//res.send("Hi friend");
+//});
+
+const todayDate = new Date();
+//future meals
+app.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex
+      .select("*")
+      .from("meal")
+      .where("meal_time", ">", todayDate);
+    if (futureMeals.lenght > 0) {
+      res.status(200).json(futureMeals);
+    }
+    res
```

Comment:
So here it is a good question if the endpoint should return an error or an empty array if it has no future meals - think about the UI that would get this data - if it is possible that we simply do not have any future meals yet, then it should be an array, if we always must have some future meals then it should be an error. Sometimes no data is actually a valid state :)